sdk/servicebus/Azure.Messaging.ServiceBus/src/Administration/SharedAccessAuthorizationRule.cs

```diff
@@ -268,9 +268,9 @@ public override bool Equals(AuthorizationRule other)
         private static string GenerateRandomKey()
         {
             byte[] key256 = new byte[32];
-            using (var rngCryptoServiceProvider = new RNGCryptoServiceProvider())
```

Comment:
See: [SYSLIB0023: RNGCryptoServiceProvider is obsolete](https://learn.microsoft.com/en-us/dotnet/fundamentals/syslib-diagnostics/syslib0023)